invenio_communities/assets/semantic-ui/js/invenio_communities/members/api/config.js

```diff
@@ -0,0 +1,43 @@
+// This file is part of Invenio-communities
+// Copyright (C) 2022 CERN.
+//
+// Invenio-communities is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+import axios from "axios";
+
+const apiConfig = {
```

Comment:
very similar code to the other `api.js`, which should be also refactored (ask me)